wet-boew/geomap/geomap-gc-chckbxrdio-fr.html

```diff
@@ -0,0 +1,203 @@
+---
+{
+	"title": "Géocarte avec grandes cases à cocher",
+	"language": "fr",
+	"description": "Composant Géocarte avec de grandes cases à cocher pour une meilleure accessibilité.",
+	"altLangPage": "geomap-gc-chckbxrdio-en.html",
+	"dateModified": "2025-11-12"
+}
+---
+
+<section>
+	<!-- GeomapStart -->
+	<div id="sample_map"
+		class="wb-geomap position scaleline geocoder geolocation large-checkboxes"
+		data-wb-geomap='{
+			"tables": [
+				{
+					"id": "bbox",
+					"zoom": true,
+					"popups": true
+				},
+				{
+					"id": "cities",
+					"zoom": true,
+					"tooltips": true,
+					"tooltipText": "Population 2011",
+					"popups": true,
+					"visible": true,
+					"popupsInfo": {
+						"content": "<div style=\"white-space:nowrap;\"><strong>Subdivision de recensement&nbsp;: </strong>_Census subdivision<br><strong>Population&nbsp;: </strong>_Population 2011</div>"
+					},
+					"style": {
+						"type": "rule",
+						"rule": [
+						{
+							"field": "Rang",
+							"value": [3],
+							"filter": "GREATER_THAN",
+							"init": {
+								"fillColor": "#e74c3c",
+								"pointRadius": 2.6,
+								"fillOpacity": 1.0,
+								"strokeWidth": 8,
+								"strokeColor": [ 231, 76, 60, 0.5 ]
+							}
+						},
+						{
+							"field": "Rang",
+							"value": [2, 3],
+							"filter": "BETWEEN",
+							"init": {
+								"fillColor": "#e74c3c",
+								"pointRadius": 4,
+								"fillOpacity": 1.0,
+								"strokeWidth": 12,
+								"strokeColor": [ 231, 76, 60, 0.5 ]
+							}
+						},
+						{
+							"field": "Rang",
+							"value": [1],
+							"filter": "EQUAL_TO",
+							"init": {
+								"fillColor": "#e74c3c",
+								"pointRadius": 5.3,
+								"fillOpacity": 1.0,
+								"strokeWidth": 16,
+								"strokeColor": [ 231, 76, 60, 0.5 ]
+							}
+						}]
+					}
+				}
+			]
+		}'>
+
+		<div class="row">
+			<div class="col-md-9 form-group">
+				<!-- Insert Map Start (mandatory) -->
+				<div class="wb-geomap-map"></div>
+				<!-- Insert Map End -->
+			</div>
+
+			<!-- Insert Legend Start (optional) -->
+			<section class="col-md-3 form-group">
+				<div class="panel panel-default">
+					<header class="panel-heading">
+						<h3 class="panel-title">Légende</h3>
```

Comment:
```suggestion
						<h2 class="panel-title">Légende</h2>
```

or you can add an H2 before the map